ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunnerMetrics.java

```diff
@@ -98,13 +114,19 @@ public class CommandRunnerMetrics implements Closeable {
         (config, now) -> commandRunner.checkCommandRunnerStatus().name());
     this.metrics.addMetric(commandRunnerDegradedReasonMetricName, (Gauge<String>)
         (config, now) -> commandRunner.getCommandRunnerDegradedReason().name());
+    this.metrics.addMetric(commandRunnerStatusNumMetricName, (Gauge<Integer>)
+            (config, now) -> commandRunner.checkCommandRunnerStatus().ordinal());
```

Comment:
Though the Enum values are unlikely to change, if it happens in future, this might break since the metric value is assumed implicitly.
We can add a documentation in these enums explaining that their ordinal values are emitted as metrics and therefore should not be changed or changed carefully to maintain compatibility.
OR add an integer field representing the numerical value of the status in the Enum constants and use that instead of ordinal.

Reply:
Thanks @pbadani. I have added the documentation on all the enums except `State` enum as it is coming from the kafka-streams library.